### ISSUE TITLE
SEO optimization

### ISSUE THREAD

**Issue by smtheard** [smtheard/ShowTracker#31]
need to do research on this, could start with not having survivor's description as overseers metadata -.-

eg. 

![image](https://cloud.githubusercontent.com/assets/3514878/26663043/bac9c7be-464d-11e7-8e65-c439c76aef48.png)

**Comment by smtheard** [comment]
http://searchengineland.com/seo-checklist-for-startup-websites-170965

basic checklist that I need to do before closing this issue

**Comment by Toltar** [comment]
Server side rendering with React would help.

**Comment by smtheard** [comment]
Agreed, I believe if I bump the version of bottle-react I am using we can flip a switch and turn on server side rendering.
